model/src/main/java/com/epam/esm/dao/queries/TagQuery.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.esm.dao.queries;
+
+public class TagQuery {
+    public static final String GET_ALL = "SELECT * FROM tag";
+    public static final String GET_BY_ID = "SELECT * FROM tag WHERE id = ?";
+    public static final String GET_BY_NAME = "SELECT * FROM tag WHERE name = ?";
+    public static final String DELETE = "DELETE FROM tag WHERE id = ?";
+    public static final String INSERT = "INSERT INTO tag(name) VALUES(?)";
+    public static final String GET_TAGS_FOR_CERTIFICATE = "SELECT id, name FROM tag\n" +
+            "         JOIN gift_certificate_tag ON tag.id = gift_certificate_tag.tag_id\n" +
```

Comment:
> Why INNER JOIN?

We can use any join here as null values are not possible in these tables, but if null values were possible then we wanna use inner join because we need to display only existing tags what were assigned to a specific certificate